cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -167,12 +174,40 @@ public static KafkaCluster fromConfigMap(ConfigMap kafkaClusterCm) {
         if (rackConfig != null) {
             kafka.setRackConfig(rackConfig);
         }
-        kafka.setInitImage(Utils.getNonEmptyString(data, KEY_INIT_IMAGE, DEFAULT_INIT_IMAGE));
+        kafka.setInitImage(Utils.getNonEmptyString(data, KEY_INIT_IMAGE, Kafka.DEFAULT_INIT_IMAGE));
         kafka.setUserAffinity(Utils.getAffinity(data.get(KEY_AFFINITY)));
 
         return kafka;
     }
 
+    public static KafkaCluster fromCrd(KafkaAssembly kafkaAssembly) {
```

Comment:
Javadoc

Reply:
Yeah, thanks! I also intend to change the name at some point because really CRD is an impl detail. The natural name might be `fromAssembly`, but we have one of those already. Maybe `fromResource()`, or `fromDesired()`?